packages/aws-client/src/S3ObjectStore.ts

```diff
@@ -18,7 +19,9 @@ import awsClient from './client';
 
 const log = new Logger({ sender: '@cumulus/aws-client/S3ObjectStore' });
 
-type QueryParams = { [key: string]: number | string };
+type QueryParams = { [key: string]: string };
```

Comment:
query params should have string value, otherwise, we will get `SignatureDoesNotMatch` when access the s3 presigned url.
localstack doesn't detect this type of `SignatureDoesNotMatch` error unfortunately.